client/allocrunner/taskrunner/plugin_supervisor_hook.go

```diff
@@ -203,16 +203,23 @@ func (h *csiPluginSupervisorHook) ensureSupervisorLoop(ctx context.Context) {
 	socketPath := filepath.Join(h.mountPoint, structs.CSISocketName)
 	t := time.NewTimer(0)
 
+	var client csi.CSIPlugin
```

Comment:
Bah, I got burned by the classic "nil interface" bug. Need to fix before I can make this ready for review.